german/finger-ins-loch/finger-ins-loch.md

```diff
@@ -5,3 +5,11 @@ Ihnen sticht eine Kiste ins Auge. Sie öffnen sie,
 finden darin einen Lappen, ziehen den Finger 
 aus dem Loch und stopfen den Lappen ins Loch. 
 Nun müssen Sie schnellstmöglichst ans Ufer.
+
+Dazu …
+
+… springen Sie [ins Wasser](ins-Wasser/ins-Wasser.md) und schieben das Boot an Land.
```

Comment:
This link will start in the `finger-ins-loch` directory (since that's where this file is). You may want to move your other directories into this directory. That would improve the organization, and I believe it would make these links work.